internal/cli/iam/projects/list.go

```diff
@@ -76,6 +79,21 @@ func ListBuilder() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return opts.Run()
 		},
+		PostRun: func(cmd *cobra.Command, args []string) {
```

Comment:
is the end goal to have each command implement this? could we instead move this to the already existing logic for `PostRun` of the root command?

Reply:
I'm not sure exactly how it will pan out in the end - it will depend on what data we can get where, eg: orgId and projectId seem to be available only in the leaf command's GlobalOpts (but we might be able to obtain that server-side instead...).

Also, we cannot put it in the root command's PostRun as only the leaf command's PostRun gets called.

We can put it in the root command's PersistentPostRun though, but we need to be careful because if a PersistentPostRun is later added to a leaf command then the root command's PersistentPostRun will not be called...

So, for now I will move the logic out of the projects list leaf command's PostRun to a more generic TrackCommand function in telemetry.go, and call it from rootCmd's PersistentPostRun. We can refactor as needed as the project progresses - in the end it would be good if we can avoid having to add tracking logic to every single leaf command.